v2/apiserver/internal/core/events.go

```diff
@@ -403,13 +406,21 @@ func (e *eventsService) createSingleEvent(
 
 	event.ID = uuid.NewV4().String()
 
+	jobs := []Job{}
 	workerSpec := project.Spec.WorkerTemplate
+	// If the event is a retry of another, defer to the Worker.Spec on the event
+	// itself.  Retain the Jobs slice as well.
+	if event.Labels != nil && event.Labels["retryOf"] != "" {
```

Comment:
If this is going to be a label, we might consider using a key that's less prone to collision by prefixing with `brigade.sh/`. Come to think of it, we should have done the same for the `clone` command.